public/css/header.css

```diff
@@ -86,17 +86,17 @@ nav {
   font-size: 16px;
 }
 
-.active-link {
-  font-weight: 700;
-  display: block;
+.nav-link span {
+  position: relative;
+  pointer-events: none; /* Prevents conflicts with GA events that should fire off the wrapping <a> tag. */
 }
 
-.active-link-underline {
-  position: relative;
-  pointer-events: none; /* Make sure span attributes aren't used for GA ping */
+.active-link {
+  font-weight: 700;
 }
 
-.active-link-underline::after {
+.active-link span::after,
```

Comment:
`.active-link span::after` has classes both here and on L114. Should that be the case?

Reply:
Fixed - thanks!